packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -2,20 +2,21 @@ import * as React from 'react';
 import {
   BaseComponent,
   autobind,
-  css,
+  customizable,
   getRTL
 } from '../../Utilities';
 import { FocusZone, FocusZoneDirection } from '../../FocusZone';
 import { Link } from '../../Link';
 import { Icon } from '../../Icon';
-import { IconButton } from '../../Button';
-import { IBreadcrumbProps, IBreadcrumbItem } from './Breadcrumb.Props';
+import { CommandButton } from '../../Button';
```

Comment:
CommandButton is deprecated, use ActionButton

Reply:
fixed